config/clusters/carbonplan/common.values.yaml

```diff
@@ -146,6 +149,10 @@ basehub:
           allowed_users: &users
             - jhamman
           admin_users: *users
+        # Temporarily increase timeout to debug #2271
+        # TODO(pnasrat): remove
+        KubeSpawner:
+          http_timeout: 600
```

Comment:
The default of this is 120, set in basehub with the note:

```
        # Increase timeout for Jupyter server to become 'ready', until
        # https://github.com/2i2c-org/infrastructure/issues/2047 is fixed
        http_timeout: 120
```

The issue https://github.com/2i2c-org/infrastructure/issues/2047 is not closed yet, so this can make sense. I'm not confident on the details about this but I think this relates to individual requests kubespawner does, and it will make request against the k8s api-server overall.